### PR TITLE
COM-465 Fix HIV adult form translation issues

### PR DESCRIPTION
**Pull request by lauravignoli**

### CODE REVIEW

File: configuration/concepts/1000_adult_hiv_initial_form.csv [hunk 1]
```diff
@@ -223,25 +223,25 @@ a048b263-c12e-4e48-b1eb-1f9a7d11153e,,Normal;Abnormal,Examen digestif,Gastrointe
...
-234fb6cd-2998-4c2e-ae9e-63acc0745a98,,Yes full name;No full name,Prescription ARV prescrit,Prescription prescribed ARV,Prescription ARV prescrit,Prescription prescribed ARV,Misc,Coded,,
+d048b999-a723-40af-a0a2-935d6e61c35c,,,"Not ApplicaIndiquez l'élément classant au stade OMS (par exemple : si : Stade I, préciser (Asymptomatique ou Lymphadènopathie généralisée persistante)ble","Indicate the classifying element at WHO stage (for example: if: Stage I, specify (Asymptomatic? Persistent generalized lymphadenopathy?) ","Not ApplicaIndiquez l'élément classant au stade OMS (par exemple : si : Stade I, préciser (Asymptomatique ou Lymphadènopathie généralisée persistante)ble","Indicate the classifying element at WHO stage (for example: if: Stage I, specify (Asymptomatic? Persistent generalized lymphadenopathy?)",Misc,Text,,
```
Comment (tmvumbi2) [inline configuration/concepts/1000_adult_hiv_initial_form.csv]: @lauravignoli Looks like a copy/paste typo: see `Not Applica` in the 2 french translations: `Not ApplicaIndiquez l'élément classant au stade OMS ...`

**Comment by tmvumbi2** [comment]
@lauravignoli I noticed a very long list of uuids in the server log (not sure what's causing this). Note: I removed 1000_diagnosis_to_retire.csv and 1001_diagnosis.csv to speed up testing.

The concept `Signes ou symptômes évocateurs de la Tuberculose` (uuid: 469e4748-382d-446c-b967-02606d02c4f1) isn't being updated when testing.
image

Server log:

```...;a04c2330-94ea-454c-ae5e-6e821ec0faae;b88be07f-c60f-4195-90e3-a842e560dace;14fc36fe-68ae-4bf0-b750-f92a1f835075;73fb0ef6-1c53-444a-9dda-edb0ec9b3b79;62515d3b-4e99-4472-b2f7-3a39cc76d1fa;928502b3-138f-4ffd-824a-ad92d99fb409;01b2322e-0b81-4e23-9ee4-928f6bc4c24f;c445b8ef-bcfd-4630-996a-0aa31114e217;a9378f7d-727f-43da-8854-2528df097214;ef733154-73d8-4da0-bed2-536e93b4e319;e800bc99-3b42-4646-ba0a-f87ecf760f8a;4a84111e-80c8-4c00-8bd3-c77e36c0cb07;2b2468d3-ab41-4614-a9af-a6e73b7619e4;ffb5028f-ddc2-4329-aff3-7da21361c3f1;c47d2f63-b3fe-42c3-ad28-f99a879146c7;53d163af-5515-4e24-ac3d-2e3339cae343;b30202ce-e197-42c1-98fd-37a21120fc2e;84a3b53d-4b28-4f32-8e9c-f4e6d66363bc;96bacafd-1549-4fc1-95fc-160a079eb4a1, Unclassified, Unclassified, ConvSet, N/A, null, null]```

**Comment by lauravignoli** [comment]
@tmvumbi2 You need to remove 1001_diagnosis.csv and 1002_diagnosis_sets.csv

File: configuration/concepts/1000_adult_hiv_initial_form.csv (hunk 1 above)
Comment (lauravignoli) [reply configuration/concepts/1000_adult_hiv_initial_form.csv]: Thanks, will remove.

**Pull request merged**